src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -406,12 +408,161 @@ private boolean moveRubricCol(int qnNumber, int colNumber, boolean isMoveLeft) {
+        return getRubricDescriptionBox(qnNumber, subQnIndex, choiceIndex).getAttribute("value");
+    }
+
+    private boolean isRubricWeightsEnabled(int qnNumber) {
+        return browser.driver.findElement(By.id("rubricAssignWeights-" + qnNumber)).isSelected();
+    }
+
+    private boolean isRubricDescriptionBoxPresent(int qnNumber, int subQnIndex, int choiceIndex) {
+        return !browser.driver.findElements(By.id(getRubricDescriptionBoxId(qnNumber, subQnIndex, choiceIndex))).isEmpty();
+    }
+
+    public String[] getRubricColValues(int qnNumber, int choiceIndex) {
+        List<String> col = new ArrayList<>();
+
+        col.add(getRubricChoice(qnNumber, choiceIndex));
+
+        if (isRubricWeightsEnabled(qnNumber)) {
+            col.add(getRubricWeight(qnNumber, choiceIndex));
+        }
+
+        int subQnIndex = 0;
+
+        while (isRubricDescriptionBoxPresent(qnNumber, subQnIndex, choiceIndex)) {
+            col.add(getRubricDescription(qnNumber, subQnIndex, choiceIndex));
+            subQnIndex++;
+        }
+
+        return col.toArray(new String[col.size()]);
+    }
+
+    /**
+     * Checks if move left button of leftmost column is disabled.
+     * Checks if move right button of rightmost column is disabled.
+     * Checks if all other move column buttons are enabled.
+     * @param qnNumber question number.
+     * @param colIndexes indexes of the columns.
+     */
+    private void verifyRubricColumnsMovability(int qnNumber, int[] colIndexes) {
+        int leftMostColIndex = colIndexes[0];
+        int rightMostColIndex = colIndexes[colIndexes.length - 1];
+
+        // checking move buttons of leftmost column
+        assertFalse(isRubricColLeftMovable(qnNumber, leftMostColIndex));
+        assertTrue(isRubricColRightMovable(qnNumber, leftMostColIndex));
+
+        // checking move buttons of last column
+        assertTrue(isRubricColLeftMovable(qnNumber, rightMostColIndex));
+        assertFalse(isRubricColRightMovable(qnNumber, rightMostColIndex));
+
+        // checking move buttons of remaining columns
+        for (int i = 1; i < colIndexes.length - 1; i++) {
+            assertTrue(isRubricColLeftMovable(qnNumber, colIndexes[i]));
+            assertTrue(isRubricColRightMovable(qnNumber, colIndexes[i]));
+        }
+    }
+
+    /**
+     * Checks if the values given in <i>columns</i> argument match with the
+     * actual values displayed. Also checks if move left/right buttons for
+     * each column is properly enabled or disabled. Assumes that there are
+     * at least 2 columns.
+     * @param qnNumber question number.
+     * @param colIndexes indexes of the columns.
+     * @param columns column values.
+     */
+    public void verifyRubricQuestion(int qnNumber, int[] colIndexes, String[]... columns) {
+        // checking rubric column values - choice, weight, descriptions
+        int i = 0;
+        for (int colIndex : colIndexes) {
```

Comment:
It may be more intuitive to use a counter in the for loop in this case, so that it is easy to see that you are accessing the same index for both arrays.